webapp/advantage/views.py

```diff
@@ -121,9 +123,14 @@ def advantage_view(**kwargs):
                 }
 
         for contract in account["contracts"]:
-            contract_id = contract["contractInfo"]["id"]
-            contract["token"] = advantage.get_contract_token(contract_id)
             contract_info = contract["contractInfo"]
+            if not contract_info.get("items"):
```

Comment:
I am not sure I understand what this `if` or `TODO` is about.
..and is this work to be done on our end or on your end?

If this is something we have to do, I'd rather have an issue created for it and have the whole `if` removed.

Reply:
Our end. We might have some leftover contracts with no items, and therefore without EffectiveFrom, EffectiveTo etc, leading to key errors in the code below. This check makes the frontend ignore such contracts.

Reply:
Could we have the API not return contracts with no items instead?

Reply:
there should not be contracts with no items, hence the TODO for me. Once fixed on ua-contracts we can remove this check.